internal/servicehash/servicehash.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package servicehash
```

Comment:
Since this is only used in the `ddtrace/tracer` package, why did it warrant a separate package?

Reply:
This part was written by me.
@ajgajg1134  Feel free to update the package name.

Reply:
I just meant it can be put inside `ddtrace/tracer` and doesn't need a separate package because nobody else is using it.

Reply:
👍 I've moved it into the ddtrace/tracer package. If we ever want to use it elsewhere we can always just move it back out to a shared place